packages/test/snapshots/src/replayMultipleFiles.ts

```diff
@@ -74,10 +75,12 @@ class ConcurrencyLimiter {
 			assert(this.limit >= 0);
 		}
 
-		const p = worker().then(() => {
+		const p = worker().finally(() => {
```

Comment:
this seems like the kind of change that should have a unit test, since if it breaks, it could do so in a way that silently hides errors.

As I see no changes to the tests in this package, I assume this change does not have unit test coverage. Such tests could also help document the intended behavior and usage.